scripts/release.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+#
+# Name::        release.sh
+# Description:: Use this script to prepare a new release on Github,
+#               the automation will create a GH tag like 'v0.1.0'
+#               (using the VERSION file)
+# Author::      Salim Afiune Maya (<afiune@lacework.net>)
+#
+set -eou pipefail
+
+readonly org_name=lacework
+readonly project_name=terraform-ecs-agent
+readonly git_user="Alan Nix"
+readonly git_email="alan.nix@lacework.net"
```

Comment:
This is a no-no. We worked on adopting a more formal way of doing releases with https://lacework.atlassian.net/browse/ALLY-328 

From now on, we must use the [Lacework Github Account](https://github.com/lacework-releng) to do releases, we no longer use our personal accounts.

Look at:
https://github.com/lacework/terraform-aws-ecr/blob/main/scripts/release.sh#L13-L14

```suggestion
readonly git_user="Lacework Inc."
readonly git_email="ops+releng@lacework.net"
```